data/hacker_school/hacker_school_2425_1.yml

```diff
@@ -1,10 +1,10 @@
 events:
   - topic: Introduction to Electronic Music Production
-    venue: TBC
+    venue: <a href="https://www.google.com/maps/search/?api=1&query=1.2949252184799644%2C103.77431055530907">Seminar Room 12, COM3-01-21, NUS</a>
     date: 2024-08-22T18:30:00
 
   # - topic: TBC
```

Comment:
While you're at it, help add the details for W3 HS? https://t.me/c/1381486244/6543

Reply:
Done.